docs/staging/dmb/dmb-aspects-of-a-good-application.md

```diff
@@ -0,0 +1,120 @@
+
+## Endorsements and application materials
+
+Applicants must follow the {ref}`DMB application process <dmb-application>` to collect their materials on an application page.
+The application must contain evidence of appropriate work for the level of upload rights applying for.
+Some aspects are best presented with evidence of the actual action like several sponsored uploads, others are more about learning and understanding which are harder to prove.
+But if you can also provide evidence of what you've learned about these topics, from your contributon to improve documentation/process to exerpts of insights you have had along the way.
+An application pages shall include endorsements from current Ubuntu members, developers, and uploaders.
+When applying for a specific level, having endorsements from current members with the same or wider rights is required.
+
+It is recommended to have endorsements of members from outside your team/Organization. They provide implicit evidence of you being well connected in the community, rather than working in a silo.
+Depending on the upload rights being pursued, and breadth of delegated power, this is more strict and those really should have endorsements from outside their current team and, if possible, their company.
+To allow that to accrue while working towards upload rights, consider sometimes asking non-team or non-company members to review and sponsor your work.
+Also partial endorsements (see below) can help to more quickly get at least some external endorsement.
+
+Including endorsements from sponsors is the best way to show evidence.
+If an application lacks endorsements from major package sponsors, it will negatively impact an application if it causes the suspicion that those that have seen a lot of your work would not support your application.
+That does not mean, if you have many sponsors, that all of them have to endorse you, but help us to understand why some have been left out.
+
+Example:
+
+* Evelyn is a member of the Canonical Foundations Toolchain Squad. They have worked extensively on Zig, not just on the toolchain but on many leaf packages. Over time, Evelyn has built up a portfolio of helping across Ubuntu, not just with Zig but in other areas. First, they applied for a speciality packageset `zig-packages`, and got endorsements from current packageset uploaders and a Ubuntu core developer, all within Canonical Foundations (but including a Ubuntu core developer outside their immediate team). When applying for Ubuntu core developer, they went through their uploads and asked sponsors from Debcrafters and Server to provide endorsements as well. This shows a work across immediate Canonical Teams.
```

Comment:
Grammar: "shows a work"

Reply:
Thanks for raising this - this was also a bit unclear still.
Changed it to what it really is:
"This demonstrates work and interaction within, but also outside their immediate team"